README.md

```diff
@@ -496,7 +496,10 @@ We recommend querying using payment key hashes (`addr_vkh`) when possible (other
   Output
 
   ```js
-  // current supplies of given assets
-  Array<number>
+  {
+    // current supplies of given assets.
+    // entry for an asset is null if it is not found.
+    supplies: Array<number | null>
```

Comment:
I should've mentioned this in the previous PR, but I realized it just now: the response from this endpoint should also contain the asset identifier, otherwise we just have an array of numbers and don't know the specific balance for each of the assets we sent in the request.

The request identifier is just `${asset.policy}.${asset.name}`.

So the response should be in the format:
```
{
  supplies: {
    "{asset.policy}.{asset.name}": number
  }
}
```

Reply:
fixed in 3f6c0bd2d9d325c7e06db0f3d853012c6a8d4d5f